src/content/docs/style-guide/word-choice/product-capability-feature-usage.mdx

```diff
@@ -0,0 +1,199 @@
+            <td>First mention: Nerdpack® (printed assets) or Nerdpack (digital assets)
+                Subsequent mentions: Nerdpack</td>
+            <td>nerdpack
+                NerdPack
+                Nerd Pack
+                nerd pack</td>
+        </tr>
+        <tr>
+            <td>NerdStorage*</td>
+            <td>component that make up New Relic One apps; used to store and retrieve simple sets of data</td>
+            <td>First mention: NerdStorage® (printed assets) or NerdStorage (digital assets)
+                Subsequent mentions: NerdStorage
+            </td>
+            <td>Nerdstorage
+                nerdstorage
+                Nerd Storage
+                Nerd storage
+                nerd storage
+            </td>
+        </tr>
+        <tr>
+            <td>New Relic CodeStream</td>
+            <td>IDE extension that integrates with New Relic One</td>
+            <td>New Relic CodeStream (for the New Relic integration with CodeStream) or CodeStream (for just the CodeStream app)</td>
+            <td>New Relic CodeStream’s
+                New Relic Code Stream
+                Code Stream
+            </td>
+        </tr>
+        <tr>
+            <td>New Relic Edge with Infinite Tracing*</td>
+            <td>our fully-managed, tail-based, distributed tracing solution</td>
+            <td>First use: New Relic Edge with Infinite Tracing® (printed assets) or New Relic Edge with Infinite Tracing (digital assets)
+                Subsequent uses: Infinite Tracing
+            </td>
+            <td>Infinite tracing
+                infinite tracing
+            </td>
+        </tr>
+        <tr>
+            <td>New Relic Explorer</td>
+            <td>capability of New Relic One</td>
+            <td>First use: New Relic explorer
+                Subsequent uses: New Relic Explorer
+                Describing actions in the UI: Explorer"</td>
+            <td>New Relic Explorer’s
+                Explorer (okay when directing what to select in the UI)
+                explorer
+            </td>
+        </tr>
+        <tr>
+            <td>New Relic Instant Observability</td>
+            <td>ecosystem of quickstarts for New Relic One</td>
+            <td>First use: New Relic Instant Observability or New Relic Instant Observability (I/O)
+                Subsequent uses: Instant Observability or New Relic I/O (avoid using the acronym externally, if possible)</td>
+            <td>New Relic instant observability
+                instant observability
+                NRIO
+                IO
+                I/O</td>
+        </tr>
+        <tr>
+            <td>New Relic Lookout</td>
+            <td>capability of New Relic One</td>
+            <td>"First use: New Relic Lookout
+                Subsequent uses: New Relic Lookout
+                Describing actions in the UI: Lookout</td>
+            <td>New Relic Lookout’s
```

Comment:
i'd recommend replacing these less common slanted apostrophes with single quote. if you're using VSCode it highlights them with a little light bulb to let you know.

Reply:
done.